packages/react/src/tooltip/trigger/TooltipTrigger.tsx

```diff
@@ -18,13 +19,14 @@ export const TooltipTrigger = React.forwardRef(function TooltipTrigger(
   const { className, render, ...elementProps } = componentProps;
 
   const { open, setTriggerElement, triggerProps } = useTooltipRootContext();
+  const trackCursor = useTooltipTrackCursorAxisContext();
 
   const state: TooltipTrigger.State = React.useMemo(() => ({ open }), [open]);
 
   const element = useRenderElement('button', componentProps, {
     state,
     ref: [forwardedRef, setTriggerElement],
-    props: [triggerProps, elementProps],
+    props: [triggerProps, trackCursor.elementProps?.reference, elementProps],
```

Comment:
We can bypass the cost (runtime + bundle) of Floating UI's `useInteractions` hook if we pass in the element props manually. This will require 5+ sets of props to be merged though @romgrk

Reply:
uRE already uses the variadic `mergeProps`. It's actually the only place in the codebase that does.